### PR TITLE
Update docs to reflect current code base.

### PR DESCRIPTION
**Pull request by eyakubovich**
- Reformat to use one sentence per line.
- Update the text to reflect today's code base
- No documentation of client/server and multi network options yet.
That will be added once the code settles a bit.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,31 +1,26 @@
 # flannel
 
-flannel (originally [rudder](http://comments.gmane.org/gmane.linux.coreos.devel/1683)) is an overlay network that gives a subnet to each machine for use with
-Kubernetes.
+flannel (originally [rudder](http://comments.gmane.org/gmane.linux.coreos.devel/1683)) is a virtual network that gives a subnet to each host for use with container runtimes.
```
Comment (jonboulle) [inline README.md]: do we even need the rudder reference any more? flannel has gained its own reputation :-)

File: README.md [hunk 2]
```diff
@@ -1,31 +1,26 @@
...
 
-In Kubernetes every machine in the cluster is assigned a full subnet. The machine A
-and B might have 10.0.1.0/24 and 10.0.2.0/24 respectively. The advantage of
-this model is that it reduces the complexity of doing port mapping. The
-disadvantage is that the only cloud provider that can do this is GCE.
+Platforms like Google Kubernetes assume that each container (pod) has a unique, routable IP inside the cluster.
```
Comment (jonboulle) [inline README.md]: nit, Google's

File: README.md [hunk 3]
```diff
@@ -1,31 +1,26 @@
...
 ![Life of a packet](./packet-01.png)
 
 ## Building flannel
 
-* Step 1: Make sure you have Linux headers installed on your machine. On Ubuntu, run ```sudo apt-get install linux-libc-dev```. On Fedora/Redhat, run ```sudo yum install kernel-headers```.
-* Step 2: Git clone the flannel repo: ```https://github.com/coreos/flannel.git```
-* Step 3: Run the build script: ```cd flannel; ./build```
+* Step 1: Make sure you have Linux headers installed on your machine. On Ubuntu, run `sudo apt-get install linux-libc-dev`.
+On Fedora/Redhat, run `sudo yum install kernel-headers`.
+* Step 2: Git clone the flannel repo: `https://github.com/coreos/flannel.git`
```
Comment (jonboulle) [inline README.md]: shouldn't this have a `git clone` in it?

**Comment by jonboulle** [comment]
lgtm

File: README.md (hunk 1 above)
Comment (eyakubovich) [inline README.md]: @jonboulle sentimental reasons?

**Pull request merged**